src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -398,19 +388,26 @@ function BaseSelectionList({
             maxIndex={flattenedSections.allOptions.length - 1}
             onFocusedIndexChanged={updateAndScrollToFocusedIndex}
         >
-            {/* <View style={[styles.flex1, !isKeyboardShown && safeAreaPaddingBottomStyle, wrapperStyle]}> */}
             <SafeAreaConsumer>
                 {({safeAreaPaddingBottomStyle}) => (
-                    <View style={[styles.flex1, !isKeyboardShown && safeAreaPaddingBottomStyle, StyleUtils.parseStyleAsArray(containerStyle)]}>
+                    <View style={[styles.flex1, !isKeyboardShown && safeAreaPaddingBottomStyle, containerStyle]}>
                         {shouldShowTextInput && (
                             <View style={[styles.ph5, styles.pb3]}>
                                 <TextInput
-                                    ref={(el) => {
-                                        if (inputRef) {
-                                            // eslint-disable-next-line no-param-reassign
-                                            inputRef.current = el;
+                                    ref={(element) => {
+                                        textInputRef.current = element as RNTextInput;
```

Comment:
why this was added?

Reply:
You're right - it is not needed.
Removed